WordPress/Classes/ViewRelated/Menus/MenuDetailsViewController.m

```diff
@@ -156,7 +157,11 @@ - (void)showTextFieldEditingState
 
         self.doneButton.alpha = 1.0;
         self.textFieldDesignIcon.hidden = YES;
-        self.textFieldDesignView.backgroundColor = [UIColor colorWithWhite:1.0 alpha:0.6];
+        if (@available(iOS 13, *)) {
```

Comment:
Let's define a color that encapsulates this iOS version switching, like we do with other Muriel semantic colors. I think this will also need wrapping in a #XCODE11?

Reply:
Actually – in https://github.com/wordpress-mobile/WordPress-iOS/issues/12320#issuecomment-528544892, @mattmiklic recommends we update `textPlaceholder` to use `tertiaryLabel` on iOS 13. If we update that definition, we can use it here and in one or two other places in this PR you use tertiary.